lib/util/browserdetection.js

```diff
@@ -0,0 +1,45 @@
+/* globals chrome, navigator */
+'use strict';
+
+const { guessBrowser } = require('@twilio/webrtc/lib/util');
+
+function isAndroid() {
+  return /Android/.test(navigator.userAgent);
+}
+
+function isIOS() {
+  return /iPad|iPhone|iPod/.test(navigator.userAgent);
+}
+
+function isChromeIOS() {
+  return guessBrowser() === 'chrome' && isMobile() && isIOS();
+}
```

Comment:
Can we add docs to these new functions?
isAndroid
isIOS
isChromeIOS

Reply:
Hmmm, I just remembered, do we know if iPad is considered mobile? If not, isChromeIOS might return false. Also, is the workaround we're trying to do applicable for both iPad and iPhone? @makarandp0 @PikaJoyce

Reply:
We can additionally check for `CriOS` in the regex as well. I believe that iPad falls under that category. We currently have a test that runs this which passes. The second question is definitely one that @makarandp0 knows more about!

Reply:
It seems iPad falls under mobile since the user agent contains the `mobile` string. So we're good there. Base on the existing workarounds, it seems they are getting applied for both ipad and iphone. So I think your current code works for chrome. No need to change anything here.